servicetalk-concurrent-api/src/test/java/io/servicetalk/concurrent/api/PublisherFlatMapSingleTest.java

```diff
@@ -67,7 +67,7 @@
 
 public class PublisherFlatMapSingleTest {
     @Rule
-    public final Timeout timeout = new ServiceTalkTestTimeout(30, SECONDS);
+    public final Timeout timeout = new ServiceTalkTestTimeout();
```

Comment:
nit: why this change ?

Reply:
the test doesn't need a special timeout and can use the default value. we now have a higher default value for CI that didn't exist when this timeout was originally configured.